vector/src/main/java/im/vector/app/features/home/room/detail/TimelineFragment.kt

```diff
@@ -482,6 +482,8 @@ class TimelineFragment @Inject constructor(
                 RoomDetailViewEvents.StopChatEffects                     -> handleStopChatEffects()
                 is RoomDetailViewEvents.DisplayAndAcceptCall             -> acceptIncomingCall(it)
                 RoomDetailViewEvents.RoomReplacementStarted              -> handleRoomReplacement()
+                RoomDetailViewEvents.ShowLocationSharingIndicator        -> handleShowLocationSharingIndicator()
```

Comment:
To simplify the code and not to add more lines in TImelineFragment while its already huge, I would have one Event `ShowLocationSharingIndicator` instead of two: 
```

handleShowLocationSharingIndicator(show: Boolean){
    views.locationLiveStatusIndicator.isVisible = show
}
```

Reply:
Nice!